packages/entities/entities-gateway-services/src/components/GatewayServiceList.vue

```diff
@@ -27,13 +28,16 @@
       <!-- Create action -->
       <template #toolbar-button>
         <PermissionsWrapper :auth-function="() => canCreate()">
-          <KButton
-            appearance="primary"
-            icon="plus"
-            :to="config.createRoute"
-          >
-            {{ t('gateway_services.list.toolbar_actions.new_gateway_service') }}
-          </KButton>
+          <Teleport to="#kong-ui-app-page-header-action-button">
```

Comment:
Let's use 

```
<Teleport :to="'#action-buttons-'+cacheIdentifier" :disabled="!useActionOutside">
  <PermissionsWrapper :auth-function="() => canCreate()">
    <KButton
      appearance="primary"
      data-testid="toolbar-add-route"
      icon="plus"
      :to="config.createRoute"
   >
    {{ t('routes.list.toolbar_actions.new_route') }}
   </KButton>
  </PermissionsWrapper>
</Teleport>
```

I just used `#action-buttons-'+cacheIdentifier` as the `id`. Feel free to anything you like.

Reply:
Updated